src/api/trips.js

```diff
@@ -0,0 +1,29 @@
+import Http from '../http';
+import { TRIP_STATUS } from '../constants';
+
+class Trips {
+
+  static async updateTrip(tripOptions={}, status = TRIP_STATUS.UNKNOWN) {
```

Comment:
Why have the default param in this case?

Reply:
ah yeah not needed